packages/api/lambdas/ems-metadata-report.js

```diff
@@ -90,7 +90,7 @@ const instrument = (collection) =>
 const emsMapping = {
   product: (collection) => collection.ShortName,
   metaDataLongName: (collection) => collection.DataSetId,
-  productLevel: (collection) => collection.ProcessingLevelId.replace(/^Level\s?/i, ''), // optional field
+  productLevel: (collection) => collection.ProcessingLevelId.replace(/^level\s?/i, ''), // optional field
```

Comment:
Agree these two are the same - was this a rule violation?    For that matter *why* is it case insensitive?    🤔

Reply:
It was a rule violation. My guess is that, if you're doing a case-insensitive match, it's more intuitive to only use one case when defining the regex.